INSTALL.md

```diff
@@ -242,6 +242,17 @@ filling in this section of the README.
 cargo build --release
 ```
 
+On Linux, if it is desired to build Alacritty without support for either the X11
```

Comment:
```suggestion
On Linux/BSD, if it is desired to build Alacritty without support for either the X11 or Wayland
```

Reply:
I genuinely have no idea how I screwed that up, this is what I get for not copy and pasting.